examples/platform_services/android/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.gradle
```

Comment:
We shouldn't land another copy of the FlutterPlugin.  Instead, please use the new `flutter create platform_services` template's approach of referencing the central copy of the plugin.